src/EnvelopeBuilder.php

```diff
@@ -188,20 +192,39 @@ public function createEnvelope(): string
     {
         try {
             $this->validate();
+
+            $key = '[Docusign] Create document';
+            $this->stopwatch->start($key);
             $this->createDocument();
+            $this->stopwatch->stop($key);
+            unset($key);
+
             $this->addSigner($this->signerName, $this->signerEmail);
             $this->defineEnvelope();
+
+            $key = '[Docusign] Send envelope';
+            $this->stopwatch->start($key);
             $this->sendEnvelope();
+            $this->stopwatch->stop($key);
+            unset($key);
```

Comment:
In this case, set and unset right after for a 2x use is heavier than just passing the string directly.

with var: https://3v4l.org/9BQFO/vld#output (9 ops)
without var : https://3v4l.org/NEalP/vld#output (7ops)